pallets/process-validation/src/restrictions.rs

```diff
@@ -44,22 +45,28 @@ where
         metadata_key: TokenMetadataKey,
         metadata_value: TokenMetadataValue,
     },
+    FixedOutputMetadataValueType {
+        index: u32,
+        metadata_key: TokenMetadataKey,
+        metadata_value_type: TokenMetadataValueDiscriminator,
```

Comment:
does `TokenMetadataValueDiscriminator` end up as dropdown of types in the polkadot explorer?

Reply:
I don't believe so. I believe the frontend uses the supplied types to generate names. These names are all internal to the build